build fail

[ 63%] Generating documentation with Sphinx
/bin/sh: 1: SPHINX_EXECUTABLE-NOTFOUND: not found
make[2]: *** [docs/CMakeFiles/Sphinx.dir/build.make:58: docs/CMakeFiles/Sphinx] Error 127
make[1]: *** [CMakeFiles/Makefile2:185: docs/CMakeFiles/Sphinx.dir/all] Error 2
make: *** [Makefile:152: all] Error 2

I install the Sphinx.
sudo apt-get install sphinxsearch

But it doesn't work. Build still fails.

--- a/docs/CMakeLists.txt
+++ b/docs/CMakeLists.txt
@@ -31,14 +31,17 @@ add_custom_target(gattlib-doxygen ALL DEPENDS ${DOXYGEN_INDEX_FILE})
 #
 
 find_program(SPHINX_EXECUTABLE NAMES sphinx-build DOC "Path to sphinx-build executable")
- 
-set(SPHINX_SOURCE ${CMAKE_CURRENT_SOURCE_DIR})
-set(SPHINX_BUILD ${CMAKE_CURRENT_BINARY_DIR}/docs/gattlib/sphinx)
- 
-add_custom_target(Sphinx ALL
-                  COMMAND ${SPHINX_EXECUTABLE} -b html
-                  		-Dbreathe_projects.gattlib=${DOXYGEN_OUTPUT_DIR}/xml
-                  		${SPHINX_SOURCE} ${SPHINX_BUILD}
-                  DEPENDS ${DOXYGEN_INDEX_FILE}
-                  WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-                  COMMENT "Generating documentation with Sphinx")
+if (SPHINX_EXECUTABLE)
+    set(SPHINX_SOURCE ${CMAKE_CURRENT_SOURCE_DIR})
+    set(SPHINX_BUILD ${CMAKE_CURRENT_BINARY_DIR}/docs/gattlib/sphinx)
+
+    add_custom_target(Sphinx ALL
+                      COMMAND ${SPHINX_EXECUTABLE} -b html
+                              -Dbreathe_projects.gattlib=${DOXYGEN_OUTPUT_DIR}/xml
+                              ${SPHINX_SOURCE} ${SPHINX_BUILD}
+                      DEPENDS ${DOXYGEN_INDEX_FILE}
+                      WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+                      COMMENT "Generating documentation with Sphinx")
+else()
+    message("Install Sphinx to generate Python documentation")
+endif()

